---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    entando.org/entando-resource-namespace: "my-namespace"
    TestResource: "my-app"
    entando.org/entando-resource-kind: "TestResource"
  name: "my-app-23500984"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "test.org/v1"
    kind: "TestResource"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "a4f8a53a-508b-482a-9427-19ae9b600216"
action: "FAILED"
count: 1
firstTimestamp: "2021-08-11T09:36:57Z"
involvedObject:
  apiVersion: "test.org/v1"
  kind: "TestResource"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "1135834"
  uid: "a4f8a53a-508b-482a-9427-19ae9b600216"
lastTimestamp: "2021-08-11T09:36:57Z"
message: "The deployment of TestResource my-namespace/my-app failed due to Deployment\
  \ failed. Please inspect the logs of the pod my-namespace/my-app-deployment. Fix\
  \ the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "test-controller-pod"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-ource-controller"
type: "Error"
